dashboard/app/models/ability.rb

```diff
@@ -117,6 +117,9 @@ def initialize(user)
         if user.regional_partners.any?
           can [:read, :start, :end, :update, :destroy, :summary, :filter], Pd::Workshop, regional_partner_id: user.regional_partners.pluck(:id)
           can :manage_attendance, Pd::Workshop, regional_partner_id: user.regional_partners.pluck(:id), ended_at: nil
+          can :update_scholarship_info, Pd::Enrollment do |enrollment|
+            !!user.regional_partners.pluck(enrollment.workshop.regional_partner_id)
```

Comment:
I guess I'm new to this area so a bit biased towards commenting, but is this saying that only a user who has at least one enrollment in at least one regional partner's workshop that also has a regional partner (!?) is able to update a scholarship?

Reply:
`user.regional_partners` gets the [list of partners](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/user.rb#L204) that the user is a program manager for, so this is asking whether the user is a program manager for the partner whose workshop the enrollment is associated with.